capstone-ui/src/components/User/Interests/Interests.jsx

```diff
@@ -22,7 +22,7 @@ export default function Interests({userInfo, goToBasic, goToMedia, gotToEditInte
         <div className="column col-2" >
           <p className="interests-title">Movies:</p>
           {
-            userInfo.interests && userInfo.interests.movies && userInfo.interests.movies != [] ?
+            userInfo.interests && userInfo.interests.movies && userInfo.interests.movies != []  && Array.isArray(userInfo.interests.movies)?
```

Comment:
- Will there be cases where userInfo.interests.movies are not [] ?
- can youse userInfo?.interests?.movies instead of doing all "&&".
(apply to all the similar changes in this diff too like hobbies etc)